### PR TITLE
fix: [PL-42850]: changing to git clone

### PR DESCRIPTION
**Pull request by bharatgoelharness**
#### Screenshots
image (1)

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

File: src/modules/10-common/constants/TrackingConstants.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ export enum DelegateActions {
   DelegateCommandLineKubernetes = 'Delegate Command Line Kubernetes',
   DelegateCommandLineVerify = 'Delegate Command Line Verify Clicked',
   DelegateCommandLineKubernetesManifest = 'Delegate Command Line Kubernetes Manifest',
-  DelegateCommandLineKubernetesManifestCommandCopy1 = 'curl -LO https://raw.githubusercontent.com/harness/delegate-kubernetes-manifest/main/harness-delegate.yaml copied',
+  DelegateCommandLineKubernetesManifestCommandCopy1 = 'git clone https://github.com/harness/delegate-kubernetes-manifest.git copied',
```
Comment (ishantmehtaharness) [inline src/modules/10-common/constants/TrackingConstants.ts:131]: Do we need to update Docs for the same?

File: src/modules/27-platform/strings/strings.en.yaml [hunk 2]
```diff
@@ -301,7 +301,7 @@ delegates:
     firstCommandHelmSecondLine: helm repo update harness-delegate
     secondCommandHeadingHelm: Now we are ready to install the delegate
     firstComandHeadingKubernetes: Create your own YAML from a Kubernetes manifest template
-    firstComandKubernetesFirstLine: curl -LO https://raw.githubusercontent.com/harness/delegate-kubernetes-manifest/main/harness-delegate.yaml
+    firstComandKubernetesFirstLine: git clone https://github.com/harness/delegate-kubernetes-manifest.git
```
Comment (ishantmehtaharness) [inline src/modules/27-platform/strings/strings.en.yaml:304]: We can re-use this in DelegateCommandLineKubernetesManifestCommandCopy1
basically use variables and add "copied" test at the end

File: src/modules/10-common/constants/TrackingConstants.ts (hunk 1 above)
Comment (bharatgoelharness) [reply src/modules/10-common/constants/TrackingConstants.ts:131]: no

**Review by AnupamIO — APPROVED** [review]

**Comment by bharatgoelharness** [comment]
trigger messagecheck

**Review by namanharness — APPROVED** [review]

**Pull request merged**